ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/part/editor/multipart/SplitEditorPartFactory.java

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.part.editor.multipart;
+
+import com.google.gwt.user.client.ui.IsWidget;
+
+/**
+ * The factory creates instances of {@link SplitEditorPartView} to display view representation of editors
+ * in {@link EditorMultiPartStackView}.
+ *
+ * @author Roman Nikitenko
+ */
+public interface SplitEditorPartFactory {
```

Comment:
This is factory for SplitEditorPart**View** instances.
So may be it would be better to name it as SplitEditorPart**View**Factory?